boards/acd52832/src/main.rs

```diff
@@ -4,6 +4,7 @@
 // Disable this attribute when documenting, as a workaround for
 // https://github.com/rust-lang/rust/issues/62184.
 #![cfg_attr(not(doc), no_main)]
+#![feature(const_in_array_repeat_expressions)]
```

Comment:
?

Reply:
explained here: https://github.com/tock/tock/pull/1767#issuecomment-645646829

I can add a comment saying as much in every board file